src/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+# set minimum cmake version
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+
+# Uncomment for VERBOSE Makefiles. Useful for debugging.
+set(CMAKE_VERBOSE_MAKEFILE ON)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
given the comment above, I guess you want to comment these lines? :)

Reply:
I will comment out the `CMAKE_VERBOSE_MAKEFILE`.
I need `CMAKE_INCLUDE_CURRENT_DIR` as I create the `mpif.h` on the current dir if this is not a MPI build.

Reply:
Oh, you can do that more gently like this:
- generate `mpif.h` in `${ImpactT_BINARY_DIR}`
- link it via
```cmake
target_include_directories(${OUTPUT_NAME} PRIVATE $<BUILD_INTERFACE:${ImpactT_BINARY_DIR}>)
```

Reply:
Awesome! Thanks for the tip! I changed the file to use this new way and removed the `CMAKE_INCLUDE_CURRENT_DIR`.